doc/ingest-proposal.md

```diff
@@ -0,0 +1,102 @@
+# Proposal for a new Ingest design
+
+## Current Problems
+
+1. Not resilient to process death.
+2. Half the logic is in storage writer the other half in a block hook.
+3. Undefined behavior if multiple publishers sync the same chain.
```

Comment:
It seems to me that would be very rare as it would mean multiple publishers are publishing for the same provider.